packages/vue-searchbox/src/components/VueSearchbox.jsx

```diff
@@ -384,7 +393,19 @@ const VueSearchbox = {
     },
     getBackgroundColor(highlightedIndex, index) {
       return highlightedIndex === index ? "#eee" : "#fff";
-    }
+    },
+    getComponent(downshiftProps={}) {
```

Comment:
@ShahAnuj2610 Why there is a mismatch here?
Use the same name which you're using in `react-searchbox`.
Do we not have a custom renderer for `vue-searchbox`?